packages/app/pages/api/redirectionUrls.ts

```diff
@@ -0,0 +1,28 @@
+import { initializeApp } from 'firebase/app';
+import { collection, doc, getDocs, getFirestore, query, where } from 'firebase/firestore/lite';
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { firebaseConfig } from '../../components/constants';
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+    switch (req.method) {
+        case "POST":
+            // Create a connection to the firebase DB.
+            const app = initializeApp(firebaseConfig);
+            const db = getFirestore(app);
+            const urlCol = collection(db, 'redirect-urls');
+            getDocs(query(urlCol, where("contractAddr", "in", req.body.addresses)))
```

Comment:
this could stay a POST bc we are using an array of addresses.

v2 of this should prob get all urls tied to a user, so that we could use a `GET`